05_run_ocp.sh

```diff
@@ -64,7 +64,7 @@ sudo cp ocp/bootstrap.ign ${IGN_FILE}
 ./get_rhcos_image.sh
 LATEST_IMAGE=$(ls -ltr redhat-coreos-maipo-*-qemu.qcow2 | tail -n1 | awk '{print $9}')
 sudo cp $LATEST_IMAGE /var/lib/libvirt/images/${CLUSTER_NAME}-bootstrap.qcow2
-virt-install --connect qemu:///system \
+sudo virt-install --connect qemu:///system \
```

Comment:
Any idea why this is needed, when all we're doing is generating the xml?  Fine to add it but weird that I don't in my env (the user is in the libvirt group)

Reply:
I didn't look to hard to be honest, my stack user wasn't in the virt group so thats probably it